server-side-scanners/boostsecurityio/sci-sca/rules.yaml

```diff
@@ -16,3 +16,14 @@ rules:
     pretty_name: Use of Unsafe AI model
     ref: https://huggingface.co/docs/hub/en/security-pickle
     recommended: true
+  forbidden-component:
+    categories:
+      - ALL
+      - boost-baseline
+      - use-of-forbidden-component
```

Comment:
@scott-boost did you update dev + prod rules-management with the new category and group?